### PR TITLE
VIH-10570 - Axe Failure: Venue-list-component

### PR DESCRIPTION
**Pull request by marcogagliardi**
[VIH-10570](https://tools.hmcts.net/jira/browse/VIH-10570)

added label tag

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/venue-list/venue-list.component.html [hunk 1]
```diff
@@ -2,30 +2,36 @@
   <div class="govuk-grid-column-full">
     <h1 class="govuk-heading-m">{{'venue-list.title' | translate}}</h1>
   </div>
+
     <div class="govuk-grid-column-three-quarters">
-        <ng-select id="venue-allocation-list" class="selection-list"
+        <label>
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/venue-list/venue-list.component.html:7]: An empty label won't fix accessibility. Ng-Select has an attribute `labelForId`. Here's a snippet of an example from the library repo. [Full example here](src/demo/app/examples/forms-async-data-example/forms-async-data-example.component.html)

```
<label for="album">Loading async data</label>
<ng-select [items]="albums"
           #select
           bindLabel="title"
           dropdownPosition="auto"
           bindValue="id"
           labelForId="album"
           placeholder="Select album"
           [virtualScroll]="true"
           formControlName="album">
    <ng-template ng-option-tmp let-item="item" let-search="searchTerm">
        <div><span>Title: </span><span [ngOptionHighlight]="search">{{item.title}}</span></div>
        <small><b>Id:</b> {{item.id}} | <b>UserId:</b> {{item.userId}}</small>
    </ng-template>
</ng-select>
```

This applies to the second ng-select instance

**Review by shaed-parkar — COMMENTED** [review]
The label element should live side by side to the element it is a label for. Or use one of the suggestions solutions https://dequeuniversity.com/rules/axe/4.1/label#:~:text=Effective%20form%20labels%20are%20required,labels%20to%20identify%20form%20fields.

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/venue-list/venue-list.component.html (hunk 1 above)
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/app/shared/venue-list/venue-list.component.html:7]: the empty label is fixing the Axe. Your fix will not work as I have already tried it. Please run the Axe tool and you will see it.

**Comment by marcogagliardi** [comment]
https://dequeuniversity.com/rules/axe/4.8/label?application=AxeChrome

**Review by shaed-parkar — APPROVED** [review]

**Pull request merged**